examples/ethers-v5/.prettierrc

```diff
@@ -0,0 +1 @@
+{ "printWidth": 100, "singleQuote": true }
```

Comment:
we already have `prettierrc` file in this project. If it's not picked up probably we need to explicitly point prettier to it. Reverting this should also avoid reforming index.ts

Reply:
It's fixed.

I accidentally just opened the examples folder in my IDE, that's why it did not pick it up.